### DISCUSSION TITLE
Question is the bold font correct

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#331] [Q&A]
Bold Text Rendering: For paragraphs, bold text looks correct in the editor (narrow AI brand font) but appears incorrectly on the frontend (regular, non-narrow font).

![image](https://github.com/user-attachments/assets/6f0aa287-21be-42ab-9db6-d12c2d5ddd2b)

Question from @amnesty-service

**Comment by jaymcp** [comment]
This is a bug. `<strong>` tags are set to Amnesty Trade Gothic Condensed in the editor stylesheet, but not in the frontend stylesheet.
